packages/vats/src/bridge.js

```diff
@@ -75,9 +75,9 @@ const prepareScopedManager = zone => {
         this.state.inboundHandler = newHandler;
       },
       setHandler(newHandler) {
+        // Idempotent to avoid conditional logic for the caller to decide
+        // whether to init or set.
         newHandler || Fail`Must provide a handler for ${this.state.bridgeId}`;
-        this.state.inboundHandler ||
-          Fail`Bridge handler not yet initialized for ${this.state.bridgeId}`;
```

Comment:
FYI this will require an upgrade of vat-bridge before any other vats.

Also I don't recall why we separated initHandler from setHandler, but there was definitely an intent that setHandler be used only for updates

Reply:
Retracted.